Products/zms/ZMSMetaobjManager.py

```diff
@@ -1142,7 +1142,21 @@ def manage_changeProperties(self, lang, btn='', key='all', REQUEST=None, RESPONS
           self.setMetaobj( newValue)
           # Change attributes.
           for old_id in REQUEST.get('old_ids', []):
-            attr_id = REQUEST['attr_id_%s'%old_id].strip()
+
+            try:
+              attr_id = REQUEST['attr_id_%s'%old_id].strip()
+            except:

```

Comment:
when can this happen?

Reply:
attr_id can be not a string but a list if id (unitentionally) occurs 2x  by multiple row posting. And strip() does not work on lists, If a list tyoe can be avoided for sure, this exception is not needed, of course.